taxi/views.py

```diff
@@ -59,6 +83,22 @@ class CarListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
     queryset = Car.objects.all().select_related("manufacturer")
 
+    def get_queryset(self):
+        model = self.request.GET.get("model")
+        queryset = Car.objects.all()
```

Comment:
since you override default queryset with get_queryset, you should remove default queryset since its overriden, also, why you removed select_related manufacturers?